engine/src/main/java/org/terasology/config/NetworkConfig.java

```diff
@@ -70,7 +72,24 @@ public void setServerPort(int serverPort) {
     }
 
     public String getServerMOTD() {
-        return serverMOTD;
+        try {
+            StringBuffer mOTD = new StringBuffer();
+
+            FileReader reader = new FileReader("motd.md");
```

Comment:
This file reader gets never closed. Also no enclude gets specified which is bad as it is then platform specific. Should be UTF-8